Tensile/Tests/extended/stream_k/sk_sgemm.yaml

```diff
@@ -0,0 +1,116 @@
+
+GlobalParameters:
+  NumElementsToValidate: -1
+  BoundsCheck: True
+  KernelTime: True
+  DataInitTypeAlpha: 1
+  DataInitTypeBeta: 0
+  # DataInitTypeA: 1
+  # DataInitTypeB: 1
+  # DataInitTypeC: 1
+  # ValidationPrintValids: True
+  MaxWorkspaceSize: 134217728
+  # PrintSolutionRejectionReason: True
+  # ForceGenerateKernel: True
+  # GenerateSourcesAndExit: True
+  # NumWarmups: 2
+  # EnqueuesPerSync: 10
+  # NumBenchmarks: 10
+  # BufferOffsetA: 500
+  # BufferOffsetB: 700
+  # BufferOffsetC: 900
+  # BufferOffsetD: 1100
+
+BenchmarkProblems:
+  - # sgemm NT
+    - # ProblemType
+      OperationType: GEMM
+      DataType: s
+      TransposeA: False
+      TransposeB: True
+      UseBeta: False
+      Batched: True
+
+    - # BenchmarkProblemSizeGroup - Standard
+      InitialSolutionParameters:
+      BenchmarkCommonParameters:
+        - KernelLanguage: ["Assembly"]
+        - EdgeType: ["ShiftPtr"]
+        - PrefetchLocalRead: [True]
+      ForkParameters:
+        - MatrixInstruction:
+          - [32, 32, 2, 1, 1, 2,2, 2,2]
```

Comment:
MI32x32 kernels are all rejected.
We need to add more values either in DepthU or PrefetchLocalRead.
  PrefetchLocalRead = 1 or (DepthU/MatrixInstK)
MI32x32x2 case, we need 1 or 5 for PrefetchLocalRead with DepthU=8.

Reply:
Fixed